src/core/server/logging/logger.test.ts

```diff
@@ -410,3 +410,21 @@ test('passes log record to appenders only if log level is supported.', () => {
     });
   }
 });
+
+test('passes log record to appender with receiveAllLevels: true, regardless if log level is supported', () => {
+  const receiveAllAppender = { append: jest.fn(), receiveAllLevels: true };
+  const warnLogger = new BaseLogger(context, LogLevel.Warn, [receiveAllAppender], factory);
+
+  warnLogger.trace('trace-message');
+  expect(receiveAllAppender.append).toHaveBeenCalledTimes(1);
+  warnLogger.debug('debug-message');
```

Comment:
NIT:
- (probably covered in other tests, but) would probably check the call argument after each (or after last call of) `expect(receiveAllAppender.append).toHaveBeenCalledTimes(1);`
- Maybe an opposite test with `receiveAllLevels : false`